core/unix/loader_linux.c

```diff
@@ -174,20 +174,19 @@ typedef struct _tcb_head_t {
 #ifdef X86
 #    define TLS_PRE_TCB_SIZE 0
 #elif defined(AARCHXX)
-/* FIXME i#1569: This may be wrong for AArch64! */
 /* Data structure to match libc pthread.
  * GDB reads some slot in TLS, which is pid/tid of pthread, so we must make sure
  * the size and member locations match to avoid gdb crash.
```

Comment:
The code here, and where these are used in privload_tls_init(), is talking about this struct layout affecting gdb use: but I don't think anyone has hit gdb issues on AArch64 so presumably that only occurs on AArch32 (and maybe RISCV since they added the same things). You hit this on private libs calling pthread routines, right? Edit this comment to include that reason to get this to match pthread?